dev/docker.sh

```diff
@@ -84,12 +81,12 @@ if [[ -z $DOCKER_PASSWORD ]]; then
 fi
 
 # Publish the image to Docker hub.
-if [ -n "$DOCKER_PASSWORD" -a -n "$DOCKER_USERNAME" -a -n "$VERSION" ]; then
+if [ -n "$DOCKER_PASSWORD" -a -n "$DOCKER_USERNAME" -a -n "$TAGS" ]; then
```

Comment:
Isn't `-n "$TAGS"` always true?

Reply:
it is, however a bit of defensive programming does not hurt in shell scripts :-)